source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,233 @@
+.. _in-use-encryption-tab:
+
+=================
+In-Use Encryption 
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   In-Use Encryption is an Enterprise/Atlas only feature. 
+
+The :guilabel:`In-Use Encryption` connection tab allows you to connect your 
+deployments using :manual:`Queryable Encryption
+</core/queryable-encryption/>`. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`In-Use Encryption` tab.
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Provide a :guilabel:`Key Vault Namespace`.
+
+            A :guilabel:`Key Vault Namespace` refers to a collection that 
+            contains all the data keys used for encryption and decryption. 
+
+            Specify a collection in which data encryption keys are stored in 
+            the format ``<db>.<collection>``.
+
+         .. step:: Select a :guilabel:`KMS Provider`.
+
+            You can select from the following Key Management Systems: 
+
+            - :ref:`Local KMS <local-kms>`
+            
+            - :ref:`AWS <aws-compass-encryption>`
+            
+            - :ref:`GCP <gcp-compass-encryption>`
+            
+            - :ref:`Azure <azure-compass-encryption>`
+            
+            - :ref:`KMIP <kmip-compass-encryption>`
+
+            .. _local-kms:
+
+            Local KMS
+            ~~~~~~~~~
+
+            You can locally manage your key as a KMS using the :guilabel:`Local KMS`
+            option.  
+
+            Click :guilabel:`Generate Random Key` to generate a 96-byte long 
+            base64-encoded string. You need this key to access encrypted data.
+
+            .. warning::
+
+               |compass-short| does not save KMS credentials by default. Copy 
+               and save the key in an external location.
+
+            .. _aws-compass-encryption:
+
+            AWS
+            ~~~
+
+            You can use :atlas:`AWS </security-aws-kms/>` to manage your keys.
+
+            Specify the following fields:
+
+            .. list-table::
+               :header-rows: 1 
+               :widths: 50 50
+
+               * - Field
```

Comment:
**Suggestion**

Add an extra table header "Required" and mark the optional items with "No" and remove the (Optional) parenthesis from the description.

Reply:
Only suggestion is to move the required field to the middle column. Field | Required |  Description.